src/gql/fragments/projectSettings/projectEventSettings.graphql

```diff
@@ -0,0 +1,107 @@
+#import "./access.graphql"
+#import "./aliases.graphql"
+#import "./general.graphql"
+#import "./githubCommitQueue.graphql"
+#import "./variables.graphql"
+#import "./plugins.graphql"
+#import "./notifications.graphql"
+#import "./patchAliases.graphql"
+#import "./virtualWorkstation.graphql"
+
+fragment projectEventSettings on ProjectEventSettings {
+  projectRef {
+    id
+    identifier
+    repoRefId
+    ...projectGeneralSettings
+    ...projectAccessSettings
+    ...projectPluginsSettings
+    ...projectNotificationSettings
+    ...projectPatchAliasSettings
+    ...projectVirtualWorkstationSettings
+
+     versionControlEnabled  
+    
+    triggers{
+      project
+      level      
+      definitionID      
+      buildVariantRegex      
+      taskRegex      
+      status      
+      dateCutoff
+      configFile      
+      generateFile      
+      command      
+      alias      
+    }     
+    periodicBuilds{
+      id
+      configFile
+      intervalHours
+      alias
+      message
+      nextRunTime
+    }
+    tracksPushEvents    
+    hidden   
+  }
+  subscriptions {
+    ...subscriptions
+  }
+  vars {
+    ...variables
+  }
+  ...projectEventGithubCommitQueue
+  aliases {
+    ...alias
+  }
+}
+
+fragment repoEventSettings on RepoEventSettings {
+  projectRef {
+    id
+    displayName
+    ...repoGeneralSettings
+    ...repoAccessSettings
+    ...repoPluginsSettings
+    ...repoNotificationSettings
+    ...repoPatchAliasSettings
+    ...repoVirtualWorkstationSettings
+
+     versionControlEnabled  
+    
+    triggers{
+      project
+      level      
+      definitionID      
+      buildVariantRegex      
+      taskRegex      
+      status      
+      dateCutoff
+      configFile      
+      generateFile      
+      command      
+      alias      
+    }     
+    periodicBuilds{
+      id
+      configFile
+      intervalHours
+      alias
+      message
+      nextRunTime
+    }
```

Comment:
You probably worked on this before #1181 was merged in, but now you can use a periodic builds fragment!